gen/setup.sh

```diff
@@ -49,4 +49,6 @@ mv dub.json.part dub.json
 
 unset IFS
 
+dub build :configure
+dub build :splice
```

Comment:
I don't think so... it's the normal way of explicitly building only a submodule.